bin/worker.js

```diff
@@ -1,9 +1,9 @@
-import { reportRequestMetric } from '../lib/influx.js'
+import { reportRequestMetrics as defaultReportRequestMetric } from '../lib/influx.js'
```

Comment:
Nitpick - feel free to ignore.

In my experience, using "default" in a function name is very vague. It's better to name the function their specifics. In this case, I would use `reportRequestMetricsToInflux`.

```js
import { reportRequestMetricsToInflux } from '../lib/influx.js'

// ...
export default {
  async fetch(request, env, ctx, { reportRequestMetrics = reportRequestMetricsToInflux } = {}) {
// ...
```